docs/contribution/code-reviews.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Code reviews
+menu: Contributors
+---
+
+# Code reviews
+
+Reviewing pull requests is a common type of contribution.
+Despite continuous integration (CI) ensuring the code does what's supposed to do,
+it's not enough.
+There are contribution traits that can't be automated:
+_design, code structure & architecture, tests quality, or typos._
+
+This document puts together traits that we should look out for when reviewing pull requests:
+
+**👀 Readability:** Does the code express its intention clearly?
+If you need to spend a bunch of time figuring out what the code does,
+the code implementation needs to be improved.
+Suggest splitting the code into smaller abstractions that are easier to understand.
+Alternative, and as a last resource,
+they can add a comment explaining the reasoning behind it.
+Ask yourself if you'd be able to understand the code in a near future,
+without any surrounding context like the pull request description.
+
+**🌱 Small pull request:** Large pull requests are hard to review and it's easier to miss out details.
+If a pull request and unmanageable,
+suggest the author to break it down.
+
+**📦 Consistency:** It's important that the changes are consistent with the rest of the project.
+Inconsistencies complicate maintenance, and therefore we should avoid them.
+If there's an approach to output messages to the user,
+or report errors,
+we should stick to that.
+If the author disagrees with the project's standards,
+suggest them to open an issue where we can discuss them further.
+
+**✅ Tests:** Tests allow changing code with confidence.
+The code on pull requests should be tested,
+and tests should be good.
+A good test is a test that consistently produces the same result and that it's easy to understand and maintain.
+Reviewers spend most of the review time in the implementation code,
+but tests are equally important because they are code too.
+
+**⚠️ Breaking changes:** Breaking changes are a bad user experience from Tuist.
+Contributions should avoid introducing breaking changes unless it's strictly necessary.
+There are many language feature that we can leverage to evolve the interface of Tuist without resorting to them.
+Whether a change is breaking or not might not be obvious.
```

Comment:
We could also suggest to check changes against the fixtures to ensure old manifests are still compatible.